v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -66,6 +67,10 @@ public Ddl scan() {
 
     for (Map.Entry<String, NavigableMap<String, Index.Builder>> tableEntry : indexes.entrySet()) {
       String tableName = tableEntry.getKey();
+      if (builder.getTable(tableName) == null) {
+        continue; // Skipping as table does not exist
```

Comment:
Why do we need to add these checks? its unclear why we would get a table name in the indexes map? The index map seems to store index_name -> index struct.

Reply:
The index query does not join with the tables. So the filters on table do not apply here when this result is returned. Same for all the other queries here.